test/classes/initializers/compilerGenerated/had_constructor.chpl

```diff
@@ -1,20 +0,0 @@
-// This test ensures that --force-initializers doesn't break constructor types
```

Comment:
So far as I recall, all the ones after this one are probably safe from that problem